docs/user-guide/api-mediation/configuration-extender-passtickets.md

```diff
@@ -121,12 +129,18 @@ Follow these steps to enable PassTicket Support specific to your ESM.
 
 <summary> Click here for command details about configuring Zowe to use PassTickets using ACF2. </summary>
 
-1. In your ESM command line interface or other security environment, define the application session key by entering the following commands, if the session key is not already defined.
+1. Issue a SHOW CLASMAP command in TSO ACF to to identity the 3 character type code associated with APPL. Replace 'APL' with the type code listed in the SHOW CLASMAP output:
+   ```acf2
```

Comment:
Is the format of the "type of code listed in the SHOW CLASMAP output" clear to the user, or should this be an example that shows the format of this property?

Reply:
yes, the command is just SHOW CLASMAP